docs/servo_power_test/src/power_test_controller.ts

```diff
@@ -15,6 +15,7 @@ export class PowerTestController {
   private totalHistogram = new TotalHistogram();
   private configList: Array<Config> = [];
   private currentConfigNum = 0;
+  private itrNum = 2;
```

Comment:
Also, maybe "itr" should be "iteration". A descriptive name is preferred!
https://google.github.io/styleguide/tsguide.html#descriptive-names